roles/keystone/tasks/main.yml

```diff
@@ -56,6 +57,12 @@
     - 5001
     - 35358
 
+- name : change keystone log file ownership
+  file: path={{ item }} mode=0644 owner=keystone group=keystone state=file
+  with_items:
+    - /var/log/keystone/keystone-all.log
+    - /var/log/keystone/keystone-manage.log
```

Comment:
Alternatively, we could adjust the permissions on the `/var/log/keystone` directory itself to be setgid, so that files created in them inherit group ownership. Although this would would miss if keystone was running as root instead of as the keystone user, so maybe the touch method is better.